src/app/view/selection.js

```diff
@@ -103,29 +190,29 @@ export default class Selection {
             if (bounds.y < ymin) {
                 ymin = bounds.y;
             }
-            if (bounds.x + bounds.width > xmax) {
-                xmax = bounds.x + bounds.width;
+            if (bounds.x > xmax) {
+                xmax = bounds.x;
             }
-            if (bounds.y + bounds.height > ymax) {
-                ymax = bounds.y + bounds.height;
+            if (bounds.y > ymax) {
+                ymax = bounds.y;
             }
         }
 
         for (let i in this.__connections) {
-            let render = Registry.currentDevice.getFeatureByID(this.__connections[i]);
-            bounds = render.bounds;
-            if (bounds.x < xmin) {
-                xmin = bounds.x;
-            }
-            if (bounds.y < ymin) {
-                ymin = bounds.y;
-            }
-            if (bounds.x + bounds.width > xmax) {
-                xmax = bounds.x + bounds.width;
-            }
-            if (bounds.y + bounds.height > ymax) {
-                ymax = bounds.y + bounds.height;
-            }
+            // let render = Registry.currentDevice.getFeatureByID(this.__connections[i]);
```

Comment:
Why is this commented out ?

Reply:
The connection class didn't have a .bounds attribute, and to have it run with just components I commented it out to be implemented later when bounds are added to connections